ext/debian/puppet.conf

```diff
@@ -0,0 +1 @@
+../../conf/puppet.conf
```

Comment:
It's possible that these are never addressed in any context related to Windows, which might be okay. We had issues at one point @jpartlow may remember related to files that were called even during Windows acceptances (setup steps).